Installation fails when `cargo install fzf-make` executed

```shell
error[E0308]: mismatched types
   --> /Users/${USER_NAME}/.cargo/registry/src/index.crates.io-6f17d22bba15001f/fzf-make-0.12.0/src/usecase/fzf_make/ui.rs:134:20
    |
134 |             .block(block),
    |              ----- ^^^^^ expected `ratatui::widgets::block::Block<'_>`, found `ratatui::widgets::Block<'_>`
    |              |
    |              arguments to this method are incorrect
    |
    = note: `ratatui::widgets::Block<'_>` and `ratatui::widgets::block::Block<'_>` have similar names, but are actually distinct types
note: `ratatui::widgets::Block<'_>` is defined in crate `ratatui`
   --> /Users/${USER_NAME}/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ratatui-0.24.0/src/widgets/block.rs:230:1
    |
230 | pub struct Block<'a> {
    | ^^^^^^^^^^^^^^^^^^^^
note: `ratatui::widgets::block::Block<'_>` is defined in crate `ratatui`
   --> /Users/${USER_NAME}/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ratatui-0.25.0/src/widgets/block.rs:233:1
    |
233 | pub struct Block<'a> {
    | ^^^^^^^^^^^^^^^^^^^^
    = note: perhaps two different versions of crate `ratatui` are being used?
note: method defined here
   --> /Users/${USER_NAME}/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tui-term-0.1.6/src/widget.rs:179:12
    |
179 |     pub fn block(mut self, block: Block<'a>) -> Self {
    |            ^^^^^

error[E0277]: the trait bound `PseudoTerminal<'_>: Widget` is not satisfied
   --> /Users/${USER_NAME}/.cargo/registry/src/index.crates.io-6f17d22bba15001f/fzf-make-0.12.0/src/usecase/fzf_make/ui.rs:132:9
    |
131 |       f.render_widget(
    |         ------------- required by a bound introduced by this call
132 | /         PseudoTerminal::new(screen)
133 | |             .cursor(tui_term::widget::Cursor::default().symbol(""))
134 | |             .block(block),
    | |_________________________^ the trait `Widget` is not implemented for `PseudoTerminal<'_>`
    |
    = help: the following other types implement trait `Widget`:
              BarChart<'a>
              ratatui::widgets::Block<'a>
              Canvas<'a, F>
              Chart<'a>
              ratatui::widgets::Clear
              Gauge<'a>
              LineGauge<'a>
              List<'a>
            and 4 others
note: required by a bound in `Frame::<'_>::render_widget`
   --> /Users/${USER_NAME}/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ratatui-0.24.0/src/terminal.rs:598:12
    |
596 |     pub fn render_widget<W>(&mut self, widget: W, area: Rect)
    |            ------------- required by a bound in this associated function
597 |     where
598 |         W: Widget,
    |            ^^^^^^ required by this bound in `Frame::<'_>::render_widget`

Some errors have detailed explanations: E0277, E0308.
For more information about an error, try `rustc --explain E0277`.
error: could not compile `fzf-make` (bin "fzf-make") due to 2 previous errors
error: failed to compile `fzf-make v0.12.0`, intermediate artifacts can be found at `/var/folders/pb/1lw3rtjs4wxd6vfkq3csqsxw0000gn/T/cargo-installZ7jk9K`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
```

--- a/README.md
+++ b/README.md
@@ -70,7 +70,7 @@ nix run nixpkgs#fzf-make
 ## OS-independent method
 ### Cargo
 ```sh
-cargo install fzf-make
+cargo install --locked fzf-make
 ```
 
 # 💡 Usage

--- a/doc/README-de.md
+++ b/doc/README-de.md
@@ -69,7 +69,7 @@ nix run nixpkgs#fzf-make
 ## OS-unabhängige Methode
 ### Cargo
 ```sh
-cargo install fzf-make
+cargo install --locked fzf-make
 ```
 
 # 💡 Nutzung

--- a/doc/README-fr.md
+++ b/doc/README-fr.md
@@ -70,7 +70,7 @@ nix run nixpkgs#fzf-make
 ## Méthode indépendante du système d'exploitation
 ### Cargo
 ```sh
-cargo install fzf-make
+cargo install --locked fzf-make
 ```
 
 # 💡 Usage

